### DISCUSSION TITLE
Problem about deploying on board

### DISCUSSION THREAD

**Discussion by Eggwhite0123** [Xilinx/finn#481] [Q&A]
Hi,
We are trying to implement our model by FINN.
But on the last step to deployed on FPGA, we couldn't get any result.
However, we could successfully generate the results using PythonSim and CPPSim but failed at RtlSim.
In addition, we also tried FINN 0.6, 0.7 version and HLS 2019.1, 2020.1 version but we all failed at RtlSim.

By the way, there are so many new Finn-Example models have been released recently such as keyword-spotting and imagenet.
Will these combined notebooks similar to the [cnv_end2end_example ](https://github.com/Xilinx/finn/blob/main/notebooks/end2end_example/bnn-pynq/cnv_end2end_example.ipynb) notebook be released recently?

**Comment by rpitonak** [comment]
Hello @Eggwhite0123 

> But on the last step to deployed on FPGA, we couldn't get any result.

Can you please elaborate on this, e.g provide logs or further details?

**Reply by Eggwhite0123** [reply]
Hi @rpitonak 
while we are running rtlsim, it shows that:
(we also tried to increase LIVENESS_THRESHOLD but it didn't work as well)
![image](https://user-images.githubusercontent.com/96819514/147725425-13c2daad-5678-4dce-a76e-af150f2512d5.png)
verify on board, it just hang there without generaing any result :
![image](https://user-images.githubusercontent.com/96819514/147725485-3f0ee72c-582a-4f84-8ab5-0aaf72098d85.png)
verify cppsim, it could run successfully and have accuracy:
![image](https://user-images.githubusercontent.com/96819514/147726167-6384de5a-e41d-4b21-9197-b829f34320ae.png)
our onnx model has the following topology for the RTLsim :
![forrtlsim](https://user-images.githubusercontent.com/96819514/147727110-62d68f71-17d4-4cdf-8a8e-b572341e5a37.png)
